lib/Backends/CPU/CPULLVMIRGen.h

```diff
@@ -29,7 +29,8 @@ class CPULLVMIRGen : public LLVMIRGen {
   virtual ~CPULLVMIRGen() = default;
   /// Ctor.
   explicit CPULLVMIRGen(const IRFunction *M, AllocationsInfo &allocationsInfo,
-                        std::string mainEntryName, llvm::StringRef libjitBC);
+                        std::string mainEntryName, llvm::StringRef libjitBC,
```

Comment:
Maybe it is better to define another additional constructor which takes an object registry and leave the existing one as is to avoid breaking any existing code? Existing constructor is then equivalent to calling this one with an empty object registry, right?

Reply:
Restored the old constructor for CPULLVMIRGen. Now we have both constructors.
Yes, existing constructor will leave the object registry empty.
Done.